src/deletepy/config/logging.yaml

```
@@ -0,0 +1,61 @@
+# Logging configuration for DeletePy
+# This file provides default logging configuration that can be customized
+
+version: 1
+disable_existing_loggers: false
+
+formatters:
+  console:
+    format: "%(asctime)s | %(levelname)-8s | %(name)s | %(message)s"
+    datefmt: "%Y-%m-%d %H:%M:%S"
+    class: deletepy.utils.logging_utils.ColoredFormatter
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)